init.sh

```diff
@@ -39,4 +39,7 @@ else
     sudo docker-compose pull 
     sudo docker-compose build --no-cache
     sudo docker-compose up -d
-fi
\ No newline at end of file
+fi
+
+#INTEGROMAT URL FOR CHESS PROJECT
+export URLintegromat="https://hook.integromat.com/jjat9jry4atuu9y8xswevgilkvtk4pen"
```

Comment:
We don't export ens like this. remove this line. Please read the doc carefully before you add stuffs